use std::{error::Error};

pub mod app;

fn main() -> Result<(), Box<dyn Error>> {
    Ok(())
}